src/GitHub.VisualStudio/GitHubPanePackage.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using GitHub.VisualStudio.UI;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.ComponentModelHost;
+
+namespace GitHub.VisualStudio
```

Comment:
I would not approve this change as this doesn't really solve any performance issues. In fact it seems to make things worse as it adds synchronous query of MEF to startup path for those users that have Git provider set as default source control provider.